build/attestations/provenance.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Provenance attestations
+keywords: build, attestations, provenance, slsa
+description: Provenance build attestations
+---
+
+The provenance attestations include facts about the build process, including
+details such as:
+
+- Build timestamps
+- Build parameters and environment
+- Version control metadata
+- Source code details
+- Materials (files, scripts) consumed during the build
+
+Provenance attestations follow the
+[SLSA provenance schema, version 0.2](https://slsa.dev/provenance/v0.2#schema).
+
+## Create provenance attestations
+
+To create a provenance attestation, pass the `--attest type=provenance` option
+to the `docker buildx build` command:
+
+```console
+$ docker buildx build --tag <namespace>/<image>:<version> \
+    --attest type=provenance,mode=[min,max] .
+```
+
+## Mode
+
+You can use the `mode` parameter to define the level of detail to be included in
+the provenance attestation. Supported values are `mode=min`, and `mode=max`
+(default).
+
+### Min
+
+In `min` mode, the provenance attestations include the following information:
+
+- `materials`: inputs used to create the build. Includes base image and version
+  of the Dockerfile syntax directive.
+- `invocation.configSource`: The name of the Dockerfile used to build the image.
+- `invocation.parameters.frontend`: the type of BuildKit frontend used:
+  `dockerfile.v0` or `gateway.v0`.
+- `invocation.parameters.args`: input options passed to the BuildKit frontend.
+  This includes, for example, the target stage of a multi-stage build.
+- `invocation.parameters.locals`: directories and files passed to the build
+  (build context).
+- `invocation.platform`: OS and architecture of the build platform used.
+- `metadata.buildStartedOn`: start time of the build.
+- `metadata.buildFinishedOn`: time when the build finished.
+- `metadata.reproducible`: whether the build is reproducible.
+- `metadata.completeness`: completeness for reproducibility.
+- `metadata.vcs`: version control information, repository and revision (if
+  available).
```

Comment:
Some (or all) of these descriptions are redundant, because they're described by the spec. But do we need to clarify how some of these parameters get populated? In some cases it is less obvious, like for example `invocation.parameter.args` being options for the BuildKit frontend (as opposed to `--build-arg`).